src/app/app.component.html

```diff
@@ -53,25 +53,25 @@ <h1>Grading Assistant</h1>
                     <table>    
                         <thead>    
                             <tr>    
-                                <th>Delete</th>    
+                                <th style="opacity: 0">Delete</th>    
```

Comment:
remove in-line styles and move to CSS file

Reply:
fixed this.